constants/jobStates.js

```diff
@@ -22,5 +22,12 @@ module.exports = {
 
 		return acc;
 	}, {}),
-	byName: STATES,
+	byName: Object.values(STATES).reduce((acc, state) => {
```

Comment:
Gonna revert this change since it is easier to get the constants via `STATE.complete` vs `STATE['Completed']`